ingest-app/src/cmr/ingest/api/ingest/collections.clj

```diff
@@ -0,0 +1,69 @@
+(ns cmr.ingest.api.ingest.collections
```

Comment:
Okay, this was a bigger change than expected, but everything's working again: tests pass.